pageserver/src/http/routes.rs

```diff
@@ -1171,6 +1175,109 @@ pub fn make_router(
             "/v1/tenant/:tenant_id/timeline/:timeline_id/layer/:layer_file_name",
             |r| RequestSpan(evict_timeline_layer_handler).handle(r),
         )
+        .put("/v1/disk_usage_eviction/run", |r| {
+            RequestSpan(disk_usage_eviction_run).handle(r)
+        })
+        .put(
+            "/v1/tenant/:tenant_id/break",
+            testing_api!("set tenant state to broken", handle_tenant_break),
+        )
         .get("/v1/panic", |r| RequestSpan(always_panic_handler).handle(r))
         .any(handler_404))
 }
+
```

Comment:
NIT: I would keep router configuration at the bottom of the file, so its easy to go there and jump to needed handler

Reply:
Reordered in [bbeba32](https://github.com/neondatabase/neon/pull/3809/commits/bbeba32ec04912d23f506527ba62dc07a5daccd7). Tried to get the tenant set broken close to other tenants, global one near the end.